<launch>
  <!-- R2, SE2, or SE3 are supported -->
  <arg name="sampling_space" default="R2" />

  <node pkg="differentiable_rmap" type="NodeRmapTraining" name="rmap_training"
        output="screen">
    <rosparam subst_value="true">
      sampling_space: $(arg sampling_space)
      bag_path: /tmp/rmap_sample_set_$(arg sampling_space).bag
      svm_path: /tmp/rmap_svm_model_$(arg sampling_space).libsvm
    </rosparam>
  </node>

  <node pkg="rviz" type="rviz" name="rviz"
        args="-d $(find differentiable_rmap)/rviz/RmapTraining.rviz"
        output="log" clear_params="true" />
</launch>
